cluster/gce/util.sh

```diff
@@ -1202,7 +1204,7 @@ ADVANCED_AUDIT_WEBHOOK_INITIAL_BACKOFF: $(yaml-quote "${ADVANCED_AUDIT_WEBHOOK_I
 GCE_API_ENDPOINT: $(yaml-quote "${GCE_API_ENDPOINT:-}")
 GCE_GLBC_IMAGE: $(yaml-quote "${GCE_GLBC_IMAGE:-}")
 CUSTOM_INGRESS_YAML: |
-$(echo "${CUSTOM_INGRESS_YAML:-}" | sed -e "s/'/''/g")
+${CUSTOM_INGRESS_YAML//\'/\'\'}
```

Comment:
Same question here

Reply:
To get rid of the shellcheck issues, it is either this or a `#shellcheck disable` statement since shellcheck will complain here.
Although not a big fan of Bash myself, I think if one has Bash at hand anyway, using its functionality comes natural. Using shell internals instead of external calls brings robustness/portability and speed too.

Reply:
the former version is not more readable imho, and this is definitely going to be faster and lower dependency.
imho: sed is useful when you're not storing in a variable, or doing many expressions, neither of which we need here.

I support this change.